src/EnergyPlus/WindowManager.cc

```diff
@@ -6070,7 +6070,7 @@ namespace WindowManager {
 			TInRadFr = TInRad * root_4( ( 1.0 + 0.5 * ProjCorrFrIn ) / ( 1.0 + ProjCorrFrIn ) );
 			FrameCon = SurfaceWindow( SurfNum ).FrameConductance;
 			HInRad = 0.5 * SurfaceWindow( SurfNum ).FrameEmis * sigma * pow_3( TInRadFr + SurfaceWindow( SurfNum ).FrameTempSurfIn + TKelvin );
-			HInConvFr = 0.0;
+			HInConvFr = HInConv;
```

Comment:
@nealkruis This one does definitely look wrong.  Calculations for the frame and divider projections were added [here](https://github.com/NREL/EnergyPlusArchive/commit/44454492611150b6406fbcb84ea19b9be432545c) back in 2000.  If you search down for HInConvFr, you'll see the old equations that used HInConv are replaced with ones that use HInConvFr with no intilalization of HInConvFr if there is not projection.